src/slic3r/GUI/wxExtensions.hpp

```diff
@@ -169,8 +169,8 @@ class ScalableBitmap
                     const std::string& icon_name = "",
                     const int px_cnt = 16,
                     const bool grayscale = false,
-                    const bool resize = false, // BBS: support resize by fill border
-                    const bool use_legacy_bmp = false);
+                    const bool resize = false, 
+                    const bool use_legacy_bmp = true);
```

Comment:
@Ocraftyone @Noisyfox 
FYI. I made this change.
Did a quick scan through the most UI icons, all looks correct on Windows